sigs.yml

```
@@ -0,0 +1,502 @@
+- name: Specification SIGs
+  sigs:
+  - name: 'Specification: General'
+    meeting: Every Tuesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo
+    chat:
+      - type: slack
+        name: '#otel-specification'
+        id: C01N7PP1THC
+    invites: calendar-spec-general
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG

---

I feel that spec general is somehow a special case since the TC is the main driver here. I can't decide if this means they don't need a GC liason or if this means the whole GC should act as liason here...

---

Good point. Since there is a monthly sync between GC&TC I guess this is given already?

---

it could still be useful so we have someone responsible for writing up quarterly(?) progress updates, which could include things about how spec issue triage is going, etc... but also ok if we want to leave that as a joint activity since the whole GC is going to be involved in spec triage